test/csso-test.js

```diff
@@ -13,6 +14,20 @@ exports.csso = {
 
     test.done();
   },
+  sourcemap: function (test) {
+    test.expect(2);
+
+    test.equal(
+      fs.readFileSync('test/expected/sourcemap.css', 'utf8'),
+      fs.readFileSync('tmp/sourcemap.css', 'utf8'),
+    'should create sourcemap');
+    test.equal(
+      fs.readFileSync('test/expected/sourcemap.css.map', 'utf8').replace(/(\\r)?\\n/g, eol === '\r\n' ? '\\r\\n' : '\\n'),
```

Comment:
Had to write those replacements due to the way how sourcemaps are generated. Hacky but tests pass on windows, so i hope it is not a big deal.